src/Tizen.NUI/src/public/Window/BorderWindow.cs

```diff
@@ -460,6 +460,43 @@ public BorderDirection GetDirection(float xPosition, float yPosition)
             return direction;
         }
 
+        /// <summary>
+        /// Gets position and size of the border window
+        /// </summary>
+        /// <returns>The total window size including the border area in the default window.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Rectangle RealWindowPositionSize
+        {
+            get
+            {
+                using var position = GetPosition();
+                using var size = GetRealWindowSize();
+                Rectangle ret = new Rectangle(position?.X ?? 0, position?.Y ?? 0, size?.Width ?? 0, size?.Height ?? 0);
+                return ret;
+            }
+        }
+
+        /// <summary>
+        /// Gets size of the border window
+        /// </summary>
+        /// <returns>The total window size including the border area in the default window.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Size2D RealWindowSize
+        {
+            get
+            {
+                return GetRealWindowSize();
+            }
+        }
+
+        private Size2D GetRealWindowSize()
+        {
+            using var val = new Uint16Pair(Interop.Window.GetSize(SwigCPtr), true);
+            using Size2D size = new Size2D(val.GetWidth(), val.GetHeight());
```

Comment:
I think we should not make this `size` as 'using' Because we will return this value, and user will use this thing outsize of this functin.